modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -134,7 +134,7 @@ $ oc create -f <file-name>.yaml
 $ oc get hpa hpa-resource-metrics-memory
 
 NAME                                     REFERENCE                       TARGETS         MINPODS   MAXPODS   REPLICAS   AGE
-oc get hpa hpa-resource-metrics-memory   ReplicationController/example   2441216/500Mi   1         10        1          20m
+oc get hpa hpa-resource-metrics-memory   ReplicationController/example   2441216/500m   1         10        1          20m
```

Comment:
This should be:
```
$ oc get hpa cpu-autoscale

NAME            REFERENCE                       TARGETS         MINPODS   MAXPODS   REPLICAS   AGE
cpu-autoscale   ReplicationController/example   173m/500m       1         10        1          20m
```
Notice that the `$ oc` command refers to a different name to match the yaml file's `metadata.name` field above. Also, somehow we ended up with an extra `oc get hpa` in the name column. It should just be the name, not the whole `oc` command. Finally, I switched the example's target to be a more believable value. (0.173 cores instead of 2441216 cores)